test/flavors/jsdoc/description-test.ts

```diff
@@ -34,5 +44,10 @@ tsTest("jsdoc: Correctly discovers the description in the jsdoc", t => {
 }
 \`\`\`
 This is an example`
-	);
+	]);
+	if (!allowed.has(description)) {
+		t.fail(`Expected ${inspect(description)} to be one of ${inspect(allowed)}`);
+	} else {
+		t.is(2 + 2, 4);
```

Comment:
Not quite following why there is the `else` clause?

Reply:
It was to ensure that there was always an assertion (ava expects to find one always). Simplified with t.true instead.